bitnami/minio/Chart.yaml

```
@@ -25,4 +25,4 @@ name: minio
 sources:
   - https://github.com/bitnami/bitnami-docker-minio
   - https://min.io
-version: 11.5.1
+version: 11.5.2
```

As we are adding a new feature, we should bump the chart's version to a new minor.
```suggestion
version: 11.6.0
```